Trustroots/trustroots
# Fix Map screenshot not based on real data

#### Proposed Changes

* Change the image with the map on the home page with an updated one already provided by @rumwerfer. 

#### Testing Instructions

* In the old version of the home you see the map with 13 in London region.
* In the new version you see 41. 

Fixes #1367
